roxie/ccd/ccdcontext.cpp

```diff
@@ -2317,11 +2317,13 @@ class CRoxieContextBase : implements IRoxieSlaveContext, implements ICodeContext
             }
         }
         else
-        {        
-            VStringBuffer job("%s-%s", wuid.str(), graphName);
+        {
+            CDateTime dt;
+            dt.setNow();
+            VStringBuffer job("%s-%s-%" I64F "u", wuid.str(), graphName, dt.getHash(0));
```

Comment:
Feels like the logic should be in runK8sjob to me. I'm also wondering whether (since this is a very rare/unusual situation) we would be better adding a suffix only if job creation fails? Depends how much we care about job names being easy to type/decode...